AntonRedko/HW7/task3.py

```diff
@@ -0,0 +1,14 @@
+def count_string():
```

Comment:
good job
look at this implementation
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result